endpoints/getting-started/requirements.txt

```
@@ -1,5 +1,5 @@
 Flask==3.0.3
-flask-cors==5.0.0
+flask-cors==5.0.1
```

Minor version dropped support for 3.8, so update this dependency
```suggestion
flask-cors==5.0.1; python_version >= "3.9"
flask-cors==5.0.0; python_version == "3.8"
```